currency-rates/load_tests/load.go

```diff
@@ -0,0 +1,47 @@
+package main
```

Comment:
Minor:
Maybe is better to create next structure
root/
      tests/
           integration_tests/
           load_test/
           
So your structure will be more organized

Reply:
done